include/layers/Layer.hpp

```diff
@@ -10,9 +10,29 @@
 
 namespace itlab_2023 {
 
+enum LayerType {
+  kInput,
+  kPooling,
+  kNormalization,
+  kDropout,
+  kElementWise,
+  kConvolution,
+  kFullyConnected,
+  kOutput
+};
+
 class Layer {
  public:
   Layer() = default;
+  int checkID() const { return id_; }
+  void giveID(int id1) { id_ = id1; }
+  LayerType checkType() const { return type_; }
+  void giveType(LayerType type1) { type_ = type1; }
+  virtual void run(const Tensor& input, Tensor& output) = 0;
```

Comment:
There can be multiple of input and ouput tensors

Reply:
But all the layers that I looked at did not imply the transfer of an array of tensors, but only these two. Tell me which ones they mean and it seems to me it will be easier to run the run function for them separately.

Reply:
Created an issue for that https://github.com/embedded-dev-research/itlab_2023/issues/132